fcrepo-search-api/src/main/java/org/fcrepo/search/api/SearchParameters.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.search.api;
+
+/**
+ * A pojo encapsulating the parameters of a search
+ */
+public class SearchParameters {
+    public SearchParameters(final String query) {
```

Comment:
JavaDocs explaining something about the nature of the expected `query` parameter will be helpful for whoever comes along to implement this.